utils/schema/artifact.json

```diff
@@ -0,0 +1,194 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+
+  "type": "object",
+  "properties": {
+    "quickstarts": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/quickstart" }
+    },
+    "datasources": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/datasource" }
+    },
+    "alerts": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/alert" }
+    },
+    "dashboards": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/dashboard" }
+    },
+    "datasourceIds": { "enum": [] }
+  },
+  "required": [
+    "quickstarts",
+    "datasources",
+    "alerts",
+    "dashboards",
+    "datasourceIds"
+  ],
+  "additionalProperties": false,
+
+  "$definitions": {
+    "quickstart": {
```

Comment:
We currently appear to have some quickstarts that still have the `installPlans` field so schema validation will fail. I'm not exactly sure how we want to handle that.